app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+end
```

Comment:
- [ ] Good work here also 👏 . Kindly add the show command to the `post_controller`. 
### Expected routes:
![image](https://github.com/Otegaa/blog-app/assets/50990925/288dce16-d8bf-409d-8bb2-fd06c6ee6ce6)

### Missing code:
```
class PostsController < ApplicationController
  
  def show;
   end

end
```
- Note: It's okay for the actions to be empty for now